scripts/ci/validate-metadata-icons.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const fs = require( 'fs-extra' );
+const glob = require( 'fast-glob' );
```

Comment:
Missing `fast-glob` package in `package.json`, but there is already the `glob` package. Maybe we could use `glob` instead?

Reply:
Since we have to add `table` anyway, we might as well add `fast-glob`. This is for CI, so even tiny performance improvement matters.